pkg/sql/create_table.go

```diff
@@ -134,6 +134,20 @@ func (n *createTableNode) startExec(params runParams) error {
 		privs = sqlbase.NewDefaultPrivilegeDescriptor()
 	}
 
+	if !params.extendedEvalCtx.ExecCfg.Settings.Version.IsMinSupported(cluster.Version2_0) {
```

Comment:
I would highly recommend to move this test to sqlbase.

Right now it misses ALTER TABLE ADD COLUMN

and CREATE VIEW.

Reply:
and maybe other things (I'm not sure about all the code paths)